src/manager/manager1E.c

```diff
@@ -17,7 +16,6 @@ typedef struct {
 
 enum Manager1E_State { Init, ObserveRegion };
 
-void Manager1E_Handler(Manager1E*);
```

Comment:
I wouldn't mind keeping this sort of forward declaration, what are others' thoughts on it?

Reply:
It's declared in manager.h already (without the type though, that does not exist there)
So really unsure how to handle this, but declaring it here feels redundant

Reply:
I think for the sake of cleanliness, forward declarations for statics functions should be avoided if they are redundant. They do make sense if a pointer table depends on it.

All functions exported from the TU should have forward declarations in the header file.

Reply:
TU?

Reply:
same opinion here, declare extern functions in the header and static functions only if necessary in the source file

Reply:
Anyways, my point was, it's declared here specifically because this block has *all* the forward declarations for functions in this file (all of which except that one wouldn't get used from anywhere else I think).
So it gives a nice concise overview of the contents of this file at the same time as it is establishing the forward declarations for the .rodata stuff (which this one isn't needed for).
And since it's defined in this file and you can't change the type in the header without touching this anyways, the redundancy isn't as bad imo.

Reply:
Not a big issue, I think this can remain the way it is for now and we can come back to the issue when formatting is higher priority.